database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowStoreEntryComponent.scala

```diff
@@ -98,7 +98,8 @@ trait WorkflowStoreEntryComponent {
         // The current code only writes heartbeats on initial pickup; this needs to be fixed by instrumenting workflow
         // heartbeats into Cromwell appropriately.  But in the meantime any other Cromwell's workflows will appear
         // to be abandoned if they were picked up before `heartbeatThreshold`.
-        if row.heartbeatTimestamp.isEmpty || row.heartbeatTimestamp < heartbeatThreshold
+//        _ = println(row.heartbeatTimestamp.toString + row.workflowState.toString)
```

Comment:
remove

Reply:
Missed it while cleaning up. Will remove it.